### PR TITLE
Removed forEach loop and switched to a for loop 

### PR DESCRIPTION
**Pull request by Jessica-Mach**
SMALL FIX
This PR is fixing the way the data imports. Sometimes, we were getting the petTypes importing in a different order than the data file. 

- changed our Seeder file to use a for loop instead of a forEach loop
- also updated our models folder to be lowercase

### CODE REVIEW

**Review by jackeymason — COMMENTED** [review]
This looks good. With all the issues we've had, seeing the console log of the id's being created sequentially when yarn run db:seed would be nice. console.log(resullt.rows[0] and console.log(idArr.rows[0]) at the bottom of the loops. Seeder would only be used during dev so not a concern to have the console.logs here. No required, fine as is.

File: server/db/Seeder.js [hunk 1]
```diff
@@ -15,15 +15,17 @@ class Seeder {
...
+        let query = 'INSERT INTO pet_types (type, img_url, description) VALUES ($1, $2, $3)';
+        let result = await pool.query(query, [petTypes[i].type, petTypes[i].imgURL, petTypes[i].description]);
+      }
+
+      for (let i = 0; i < adoptablePets.length; i++) {
+        let query = 'INSERT INTO adoptable_pets (name, img_url, age, vaccination_status, adoption_story, available_for_adoption, pet_type_id) VALUES ($1, $2, $3, $4, $5, $6, $7) RETURNING id;'
+        let idArr = await pool.query(query, [adoptablePets[i].name, adoptablePets[i].img_url, adoptablePets[i].age, adoptablePets[i].vaccination_status, adoptablePets[i].adoption_story, adoptablePets[i].available_for_adoption, adoptablePets[i].pet_type_id])
+      }
+      
+      pool.end();
```
Comment (safang99) [inline server/db/Seeder.js:28]: Is this causing an error if the seeder is run twice? We may be able to remove the `pool.end()` on 28 and just keep it in the catch block

File: server/db/Seeder.js [hunk 2]
```diff
@@ -15,15 +15,17 @@ class Seeder {
...
-     pool.end();
+      for (let i = 0; i < petTypes.length; i++) {
+        let query = 'INSERT INTO pet_types (type, img_url, description) VALUES ($1, $2, $3)';
+        let result = await pool.query(query, [petTypes[i].type, petTypes[i].imgURL, petTypes[i].description]);
+      }
+
+      for (let i = 0; i < adoptablePets.length; i++) {
+        let query = 'INSERT INTO adoptable_pets (name, img_url, age, vaccination_status, adoption_story, available_for_adoption, pet_type_id) VALUES ($1, $2, $3, $4, $5, $6, $7) RETURNING id;'
+        let idArr = await pool.query(query, [adoptablePets[i].name, adoptablePets[i].img_url, adoptablePets[i].age, adoptablePets[i].vaccination_status, adoptablePets[i].adoption_story, adoptablePets[i].available_for_adoption, adoptablePets[i].pet_type_id])
+      }
```
Comment (safang99) [inline server/db/Seeder.js:26]: This looks great! One thing to keep in mind when creating the `data.json` file to store the information of pet_types and adoptable_pets, I noticing that you are hardcoding the `pet_type_id` to use and were running into issues if things are not created in the correct order.  You could include an extra piece of data for each adobptale_pet that has the pet_type_name.  Then you can use that name to run a query and get the primary key of that record on the pet_types table.  

Not something you have to change, but good to be aware of.

**Review by safang99 — COMMENTED** [review]
This all looks great 👍. I just left two small comments.  One for clarification and one for something to keep in mind as things get more nested

File: server/db/Seeder.js (hunk 1 above)
Comment (Jessica-Mach) [reply server/db/Seeder.js:28]: We're not getting an error on pool.end() anymore so we'll leave as is.

File: server/db/Seeder.js (hunk 2 above)
Comment (Jessica-Mach) [reply server/db/Seeder.js:26]: Thanks! We're a little worried about time so we're not going to change this now, but we'll it in mind.

**Pull request merged**